CandidateReranking.md

```diff
@@ -325,7 +325,7 @@ A **Response** is composed of a array of `ranked_profiles`. Each profile has the
 | Param    | Type | Description
 |----------|------|--------|
 | document_id | String | Document Id associated with a candidate profile.
-| score | Double | Score as double value as returned from the **Reranker**.
+| score | double | Score as double value as returned from the **Reranker**.
```

Comment:
"as double value" is probably redundant since we have the type info right next to it. I would just put "Score as returned from...". Also maybe include a range ([0.0, 1.0] probably?) and downcase the word "Reranker"